prow/jobs/kyma/kyma-integration-gardener.yaml

```diff
@@ -240,6 +240,54 @@ presubmits: # runs on PRs
               requests:
                 memory: 1Gi
                 cpu: 400m
+    - name: pre-main-kyma-gardener-gcp-api-gateway
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-kyma-gardener-gcp-api-gateway"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-build-pr: "true"
+        preset-cluster-version: "true"
+        preset-debug-commando-oom: "true"
+        preset-gardener-gcp-kyma-integration: "true"
+        preset-kyma-cli-stable: "true"
+      optional: true
```

Comment:
will there be follow up PR to make it mandatory?

Reply:
the prow/jobs are optional by default, so the render templates tool will change this setting when executed

Reply:
yes ofc i meant to change template in a way this job will be mandatory